src/main/kotlin/net/maizegenetics/phgv2/brapi/api/ServerInfo.kt

```diff
@@ -0,0 +1,169 @@
+package net.maizegenetics.phgv2.brapi.api
+
+import com.typesafe.config.ConfigFactory
+import io.ktor.server.application.*
+import io.ktor.server.config.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import net.maizegenetics.phgv2.brapi.model.*
+import model.Metadata
+
+
+private val config = HoconApplicationConfig(ConfigFactory.load())
+
+val contactEmail = config.property("contactEmail").getString()
+
+val documentationURL = config.property("documentationURL").getString()
+
+val location = config.property("location").getString()
+
+val organizationName = config.property("organizationName").getString()
+
+val organizationURL = config.property("organizationURL").getString()
+
+val serverDescription = config.property("serverDescription").getString()
+
+val serverName = config.property("serverName").getString()
+
+
+fun Route.serverInfo() {
+
+    get("/serverinfo") {
+
+        val endpointCalls = mutableListOf(
+
+            Service(
```

Comment:
This should only have the currently available endpoints returned here.  I am fine for now to comment out all the rest but they should only be added once a new endpoint is added.  Otherwise client applications have no idea of what actually is available to be called.

Reply:
All the endpoints except those currently supported are commented out, so the return will only send those that are supported.  Do you want us to remove the commented out endpoints?

Reply:
ah ... as you requested, allelematrix is not supported but shows up.  I will change that (and the corresponding serverInfo test case that references this)